plugins/de.cognicrypt.codegenerator/src/main/resources/TaskDesc/SecureCommunication.json

```diff
@@ -164,8 +164,8 @@
 	        {
 	            "id": "0",
 	            "element": "radio",
-	            "note": "1$$$It is highly discouraged to enable support for legacy clients. This may have grave security implications.",      
-	           	"questionText": "Which clients should the server at least support?",
+	            "note": "1$$$It is highly discouraged to enable support for legacy clients. This may have grave security implications.$$$\nCogniCrypt's analysis raises warnings when this option is selected. You can however disable such warnings for legacy configurations in the CogniCrypt preferences.",      
```

Comment:
This seems incorrect. As I said, please review what you commit before doing so.

Reply:
Hi @kruegers, thanks for reviewing. would you please tell me, why is this incorrect? it adds another note to the note in codegenerator wizard, if the checkbox in preference is not checked.

Reply:
The problem is not with the note, but with the question text below: 
"Which clients *should* the server *should* at least support?"

Reply:
uh, I fixed that. maybe when I reverted the commits it came back. I will fix it.